.docker/monolog.yaml

```diff
@@ -0,0 +1,48 @@
+when@prod:
+    monolog:
+        channels: ["deprecation"]
```

Comment:
```suggestion
#        channels: ["deprecation"]
```
I would currently deactivate deprecation logs, they are full of unfixable Doctrine message.